src/Stratis.Bitcoin.IntegrationTests.Common/Stratis.Bitcoin.IntegrationTests.Common.csproj

```diff
@@ -95,6 +95,9 @@
     <None Update="ReadyData\StratisTest150NoWallet.zip">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="ReadyData\StratisXOver15K.zip">
```

Comment:
What about the 2 just above this one? (PS thanks for setting them all to the same value)

Reply:
Ahh, they don't exist anymore